examples/fit_ODNP_data.py

```diff
@@ -0,0 +1,128 @@
+"""Fitting ODNP Datasets for Ksigma
+=================================== 
+The T1(p) and E(p) integrals are stored in
+previous post processing pulled from an H5 file.
+The inverse of the T1(p) data processing is taken
+to produce the R1(p) data which is fit using the
+polyfit of the inverse of krho with two degrees of
+freedom. The R1(p) fit is then fed into the
+fitting routine for the enhancement data prior to
+normalization. The cross relaxivity is then
+calculated using the output of these fits as well
+as the sample parameters that are fed to in (e.g.,
+ppt value, and concentration).
+"""
+from pyspecdata import *
+from sympy import symbols, Symbol, latex,lambdify
+from scipy.io import loadmat
+
+data_sets = dict({"data_dir":'AG_processed_data', #directory of the dataset of table of integrals
+    "integrals_filename":"ras.h5", #h5 file that contains a series of integrals for different samples
+    "T100_filename": "T10_DI_water_230412", #mat file with polyfit water relaxation
```

Comment:
I was specifically suggesting that you have a dictionary that contains the information about the location of each data set... The dictionary would be named differently, but the keys would be the same.  Even though this is a stylistic, it's a clearly more logical way of organizing the code.

Reply:
that does make more sense okay!